run/rsync_archive/archive-is-reachable.sh

```diff
@@ -2,4 +2,4 @@
 
 ARCHIVE_HOST_NAME="$1"
 
-ping -q -w 1 -c 1 "$ARCHIVE_HOST_NAME" > /dev/null 2>&1
+ssh -q "$RSYNC_USER"@"$ARCHIVE_HOST_NAME" exit
```

Comment:
This can block for several minutes when the server isn't responding.

Reply:
Good call, I've added a setting to timeout after 1 second